rgwadmin/errors.go

```diff
@@ -0,0 +1,260 @@
+package rgwadmin
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/pkg/errors"
```

Comment:
Hum, how do you wrap errors if you don't use this package?

Reply:
with `%w` in new errors, like

```golang
eof := io.EOF
err := fmt.Errorf("end of file detected: %w", eof)

if errors.Is(err, io.EOF) {
    ...
```

See https://golang.org/pkg/errors/ for more details.

Reply:
Hum ok.

Reply:
I'll add that %w is one easy, built-in way of doing it, but not the only way. Any type that provides `Unwrap` as noted in @nixpanic  link above will work too, so you can define new "wrapped error" types as needed. There's also support for the `Is` and `As` methods that may allow you to cut down on some of the "boilerplate" methods in this pkg as well. However, we can discuss that separately as needed.

Reply:
For most of the call I decided not to wrap since I wasn't providing any useful insight (nothing more than what internal lib were doing). However, I wrapped unmarshal errors so I could print the response that could not be unmarshalled.